src/nwb_conversion_tools/datainterfaces/ophys/scanimage/scanimageimaginginterface.py

```diff
@@ -10,7 +11,16 @@
     HAVE_PIL = False
 
 from ..tiff.tiffdatainterface import TiffImagingInterface
```

Comment:
@h-mayorquin Sorry, I'm just now realizing - this is using the `TiffImagingExtractor` instead of the `ScanImageTiffImagingExtractor`; the second one is preferred since it supports memmaping of these files whereas the first one loads the entire file as an array.

Also, as a bonus, the `ScanImageTiffImagingExtractor` has an attribute `_metdata` that has that description string.

Also also, this would swap the dependency from `PIL` to `scanimagetiff` (see ROIExtractors for detail).

Reply:
Good catch, I did not notice this either.

https://github.com/catalystneuro/nwb-conversion-tools/pull/586